This patch is used to replace the need for git.  It's a work-around
for a problem described here: https://jira.mongodb.org/browse/SERVER-21317

Patch by Roel Janssen <roel@gnu.org>
--- a/version.json	1970-01-01 01:00:00.000000000 +0100
+++ b/version.json	2016-03-16 15:10:23.933578071 +0100
@@ -0,0 +1,3 @@
+{
+    "version": "3.3.8-guix-"
+}
